src/util.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * clamp ensures a number value is between a min and max value
+ *
+ * @param value. The value to clamp
+ * @param min. The minimum value
+ * @param max. The maxiumum value
+ */
+export function clamp(value: number, min: number, max: number): number {
```

Comment:
I think I'm -1 on this. I'm not sure I see the value of the function or the validation it's used in.